src/listwiz/replace.py

```diff
@@ -48,3 +44,32 @@ def replace_sublist(l, sublist, replacement):
             i += 1
 
     return result
+
+
+def replace_elements( l, replacements ):
+    """Function to replace elements of a list.
+
+    Parameters
+    ----------
+    l : list
+        List to change.
+    replacements : dict
+        Dictionary of replacements.
+    
+    Examples
+    --------
+    >>> from listwiz.replace import replace_elements
+    Replace all 2s with 20 and all 3 with 30.
```

Comment:
```suggestion

    Replace all 2s with 20 and all 3 with 30.

```

If you build the docs locally, you should see that this doesn't render well.  Adding the newlines should fix that!